frontend/lib/ui/Button/Button.module.css

```diff
@@ -78,6 +78,40 @@
     }
   }
 
+  &.tertiary {
+    --button-border-width: 0px;
+    border: none;
+    background: transparent;
+    color: var(--color-error-darkest);
+
+    &:not(:hover, :active) {
+      box-shadow: none;
+    }
+
+    &:hover {
+      background: #fef9fa;
```

Comment:
any reason this isn't a var?

Reply:
This color was not yet in our variables and I'm not sure what old and new colors are. I'll have a call with @jorisleker about this soon.

Reply:
Added the Figma colors as css variables and used those in the new button variant.